site/source/docs/debugging/Sanitizers.rst

```diff
@@ -104,3 +104,196 @@ The minimal runtime is supported by Emscripten. To use it, pass the flag
   $ node a.out.js
   ubsan: type-mismatch
   Runtime error: The application has corrupted its heap memory area (address zero)!
+
+.. _sanitizer_asan:
+
+Address Sanitizer
+=================
+
+Clang's `address sanitizer`__ (ASan) is now also available for use with
```

Comment:
Probably don't need to say `now`. It's best not to mention that this is a new feature at all, since the documentation will still be around long after the feature ceases to be new.